src/plugins/loan-group-categories.js

```diff
@@ -58,15 +58,7 @@ export default {
 					marketingName: 'US Borrowers',
 					marketingOrder: 2,
 					expLoansIds: [1084348, 2319451, 1079005]
-				},
-				{
-					value: 'disaster_relief_covid',
-					label: 'COVID-19 Coronavirus',
-					shortName: 'the global COVID—19 response',
-					marketingName: 'COVID-19',
-					marketingOrder: 7,
-					expLoansIds: [2319451, 2320833, 2320176]
-				},
+				}
```

Comment:
How will this removal affect the subscription/mg edit functionality under settings/subscriptions?

Reply:
Prob a mute point if we don't have any/many subs using this category. Just wanted to make sure it doesn't break that interface if it's the currently set category on an account...

Reply:
Yeah, curious about that as well. I think this is a good way to go about it, but it might not allow people who are already in that category to change their other settings without selecting one of the other categories. I think just removing the option from the setup page (like how you had it earlier) would be an ok solution if there are too many issues doing it this way.